### DISCUSSION TITLE
auto-pairs overriding not work in helix-git

### DISCUSSION THREAD

**Discussion by yoomaxxx** [helix-editor/helix#8146] [Troubleshooting]
### Summary

Hello, auto-pairs overriding not work with helix-git

https://github.com/helix-editor/helix/assets/21293376/c17336de-759b-4f77-8911-00c9c3fbb30e

### Reproduction Steps

 

I tried this:

1. `hx`

I expected this to happen:

```rust
println!()
```

### Helix log

<details><summary>~/.cache/helix/helix.log</summary>

```
please provide a copy of `~/.cache/helix/helix.log` here if possible, you may need to redact some of the lines
```

</details>

### Platform

Linux

### Terminal Emulator

kitty

### Helix Version

helix-git

**Comment by pascalkuthe** [comment]
you have turned `auto-pairs` off in your config. Why would you expect helix to insert them in that case?

**Reply by yoomaxxx** [reply]
Docs:

```
Additionally, this setting can be used in a language config. Unless the editor setting is false, this will override the editor config in documents with this language.
```

https://docs.helix-editor.com/configuration.html#editorauto-pairs-section

**Reply by pascalkuthe** [reply]
you missunderstadn the config. `auto-pairs = false` forcefully disables all auto-pais for all languages.

If you just want to disable the autpairs for plaintexet files (which is all that the global setting can do besides turning off autopairs everywhere) you can just use:

`auto-pairs = {}` globally

**Reply by yoomaxxx** [reply]
Ok, do you know when a new release?
